net/transmission/Portfile

```diff
@@ -93,6 +96,18 @@ destroot {
         xinstall -m 644 -W ${worksrcpath}/utils \
         transmission-remote.1 \
         ${destroot}${prefix}/share/man/man1
+        
+        xinstall -m 644 -W ${worksrcpath}/utils \
+        transmission-create.1 \
+        ${destroot}${prefix}/share/man/man1
+        
+        xinstall -m 644 -W ${worksrcpath}/utils \
+        transmission-edit.1 \
+        ${destroot}${prefix}/share/man/man1
+        
+        xinstall -m 644 -W ${worksrcpath}/utils \
+        transmission-show.1 \
+        ${destroot}${prefix}/share/man/man1
```

Comment:
Combine the last four `xinstall`s into a single one:

```
        xinstall -m 644 -W ${worksrcpath}/utils \
            transmission-create.1 transmission-edit.1 transmission-remote.1 transmission-show.1 \
            ${destroot}${prefix}/share/man/man1
```

Or combine all five into one:

```
        xinstall -m 644 -W ${worksrcpath} \
            daemon/transmission-daemon.1 \
            utils/transmission-create.1 utils/transmission-edit.1 utils/transmission-remote.1 utils/transmission-show.1 \
            ${destroot}${prefix}/share/man/man1
```